package.json

```diff
@@ -7,6 +7,7 @@
   "private": true,
   "//": "Use gulp for dev. It's cross platform.",
   "scripts": {
+    "build": "gulp build",
```

Comment:
I thought there was a case where a user would want to just build it for dev. But maybe that's not the case. I'm using it as `npm run build -- -p` in the Dockerfile. Easy fix.